recipes/crms2map/meta.yaml

```diff
@@ -0,0 +1,108 @@
+{% set name = "crms2map" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name |lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jinikeda/{{ name |lower }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 7c718dbe50dd3fe7cb2cf294418e846028126ed6a4f9c3960db92b7cfce3ae53
+
+build:
```

Comment:
This project appears to have a cli entry point. Please include it and test it:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

Reply:
Added entry point and test command. Thank you.